tests/test_config.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import unicode_literals
+
+import unittest
+import os
+from tempfile import mkdtemp
+from shutil import rmtree
+from mock import patch
+
+import bugwarrior.config as config
+
+
+import xdg
+
+class TestGetConfigPath(unittest.TestCase):
+
+    def setUp(self):
+        self.tmpdir = mkdtemp()
+        self.old_HOME = os.environ.get('HOME', None)
+        self.old_xdg_config_home = xdg.BaseDirectory.xdg_config_home
+        self.old_xdg_config_dirs = xdg.BaseDirectory.xdg_config_dirs
+        os.environ['HOME'] = self.tmpdir
+        if config.BUGWARRIORRC in os.environ:
+            del os.environ[config.BUGWARRIORRC]
```

Comment:
Should we restore the BUGWARRIORRC environment variable in tearDown just like HOME?